openstreetmap-carto.style

```diff
@@ -14,6 +14,7 @@ node,way   admin_level  text         linear
 node,way   aerialway    text         linear
 node,way   aeroway      text         polygon
 node,way   amenity      text         polygon
+node,way   attraction   text         polygon
```

Comment:
I'm not sure if and how this works. Typically we don't change this file, since it would require database reload, which is very time consuming and is rarely done on OSMF servers.

Reply:
In cases when the column is not defined when creating database, we use the hstore column with construction `tags->'foobar'`, see for example this very short PR with `icao` and `iata` value check: https://github.com/gravitystorm/openstreetmap-carto/pull/2674/files